charts/cla/templates/deployment.yaml

```diff
@@ -26,29 +26,33 @@ spec:
           env:
           - name: HOST
             value: {{ .Values.claAssistant.host | default "example.com" }}
-          - name: GITHUB_CLIENT
-            value: {{ .Values.github.client | default "CLIENT" }}
-          - name: GITHUB_USER
-            value: {{ .Values.github.user | default "USER" }}
           - name: PROTOCOL
-            value: http
-          - name: GITHUB_ADMIN_USERS
-            value: {{ .Values.github.user | default "GITHUB_ADMIN_USERS" }}
+            value: https
+          - name: GITHUB_CLIENT
+            valueFrom:
+              secretKeyRef:
+                name: '{{ template "cla.fullname" . }}-github-credentials'
+                key: client
           - name: GITHUB_SECRET
             valueFrom:
               secretKeyRef:
                 name: '{{ template "cla.fullname" . }}-github-credentials'
                 key: secret
-          - name: GITHUB_PASS
+          - name: GITHUB_USER
             valueFrom:
               secretKeyRef:
                 name: '{{ template "cla.fullname" . }}-github-credentials'
-                key: pass
+                key: user
           - name: GITHUB_TOKEN
             valueFrom:
               secretKeyRef:
                 name: '{{ template "cla.fullname" . }}-github-credentials'
                 key: token
+          - name: GITHUB_ADMIN
```

Comment:
What is this env var used for? According to https://github.com/cla-assistant/cla-assistant#supported-environment-variables there's an optional `GITHUB_ADMIN_USERS` is that it?

Reply:
yep mistype. get set by GITHUB_ADMINS in circle env.